misc/scripts/sources.list

```diff
@@ -0,0 +1,10 @@
+deb http://gb.archive.ubuntu.com/ubuntu/ focal main restricted
```

Comment:
Note: When rebasing, make sure to remove this file (I moved it).

Reply:
ok doing this now